vignettes/Quality_Control_with_pmartR.Rmd

```diff
@@ -10,28 +10,28 @@ output:
     self_contained: yes
 bibliography: bibliography.bib
 vignette: >
-  %\VignetteIndexEntry{pmartR-peptide} 
+  %\VignetteIndexEntry{pmartR-proteomics} 
   %\VignetteEngine{knitr::rmarkdown} 
   %\VignetteEncoding{UTF-8}
 ---
 
 ```{r global_options, include = FALSE}
 knitr::opts_chunk$set(warning=FALSE, fig.width = 8, fig.height = 6)
-
+options(rmarkdown.html_vignette.check_title = FALSE)
 ```
 
 ```{r logo, out.width = "100px", echo=FALSE}
 knitr::include_graphics("pmartR_logo_final.jpg")
 
 library(pmartR)
-library(pmartRdata)
+# library(pmartRdata)
```

Comment:
uncomment

Reply:
This vignette shouldn't be in the package any more; must've reappeared with PR #228. Removed.